.github/workflows/main.yml

```diff
@@ -3,58 +3,58 @@ name: "CI Multiplatform Build"
 on:
   push:
     paths-ignore:
-      - 'docs/**'
+      - "docs/**"
     branches:
-      - master
+      - v5-macos-ci
```

Comment:
This branch needs updating. The CI in this PR should work either way because the `master` branch is specified in the `pull_requests` section